models/public/yolo-v4-tiny-tf/accuracy-check.yml

```diff
@@ -0,0 +1,102 @@
+models:
```

Comment:
@dkurt please, keep an eye on this [PR](https://github.com/openvinotoolkit/open_model_zoo/pull/2219). It contains necessary changes in validation config for aligning with original repo.

Reply:
Shall I wait for merge or you can just expand for `yolo-v4-yiny`?